interface.go

```diff
@@ -67,8 +72,20 @@ type Bucket interface {
 	// have the same type as the calling bucket object.
 	Copy(context.Context, CopyOptions) error
 
-	// Remove the specified object from the bucket.
+	// Remove the specified object(s) from the bucket.
+	// RemoveMany continues on error and returns any accumulated errors.
 	Remove(context.Context, string) error
+	RemoveMany(context.Context, ...string) error
```

Comment:
does it make more sense from an api ergonomics perspective to have this be a []string rather than a variadic function? are there cases where we really want to variadic calls?

Reply:
The DeleteMany from curator is a variadic function:
https://github.com/mongodb/curator/blob/master/sthree/bucket.go#L475